catalog/MDCCatalog/MDCCatalogCollectionViewCell.swift

```diff
@@ -39,6 +39,8 @@ class MDCCatalogCollectionViewCell: UICollectionViewCell {
     contentView.addSubview(label)
     contentView.clipsToBounds = true
     contentView.addSubview(tile)
+    self.isAccessibilityElement = true
+    self.accessibilityTraits |= UIAccessibilityTraitButton
```

Comment:
Why are we making a collection view cell into a button to VO users?

Reply:
So that they know they can activate it and it will do something. Since the title is a noun, and not a verb, I don't think it's obvious that it can be activated.  By adding the `.button` trait, it's clear that this is a thing to activate and not just a label.

Reply:
I feel like a better way than disguising a cell into a button is to add an `accessibilityHint`. What do you think? 

"The hint is a brief, localized description of the result of performing an action on the element without identifying the element or the action."

https://developer.apple.com/documentation/uikit/uiaccessibilityelement/1619585-accessibilityhint?language=objc

Reply:
I think the `.button` trait is correct here.  I don't think it's really disguising, the trait isn't meant to do anything other than provide information about the element's capability.  The definition in the headers is pretty flexible:

> // Used when the element should be treated as a button.
> UIKIT_EXTERN UIAccessibilityTraits UIAccessibilityTraitButton;

In this case, I think it should be *treated as* a button.  I also think it's much faster for the user to recognize that a particular item can be activated if they get the "button" trait rather than listening to a hint.

Reply:
In that case, do you think we should treat all collectionViewCells as buttons? What will be in future the criteria of treating a cell as a button for a11y? Why does Apple not define a UICollectionViewCell as a button trait by default?

Reply:
> In that case, do you think we should treat all UICollectionViewCells as buttons? 
No, I don't think a cell should be treated as a button unless it is acting as if it's a button. In this case, the cells are acting like a button. They provide an image and a title and respond to a single tap.  They cannot be removed, reordered, or otherwise edited.  I think that from the user's perspective, these are effectively buttons.

> What will be in future the criteria of treating a cell as a button for a11y? 
I don't think we have a single set of criteria for treating a cell as a button, but I think a good rule of thumb is that if it could be replaced by a UIButton in all but visual appearance, then it should have the `.button` trait.

> Why does Apple not define a UICollectionViewCell as a button trait by default?
I think `UICollectionViewCell` is a pretty flexible container, and much of that content might not be interactive. In many (if not most) cases the cell itself is not a single, actionable element. A few examples where I think the button trait would not be a good fit:
* A list of attendee names at an event.
* An ordering app where each cell is a topping/condiment to be added to a food item. The cells would be selectable, but are not buttons.
* A shopping list app where each cell is a more complex view containing one or more controls.

Reply:
Great, thank you for providing these answers!